lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -10803,6 +10803,14 @@ void Emit(ParseNode *pnode, ByteCodeGenerator *byteCodeGenerator, FuncInfo *func
         {
             byteCodeGenerator->EmitSuperCall(funcInfo, pnode, fReturnValue);
         }
+        else if (pnode->sxCall.pnodeTarget->nop == knopImport)
+        {
+            ParseNodePtr args = pnode->sxCall.pnodeArgs;
+            Emit(args, byteCodeGenerator, funcInfo, false);
```

Comment:
Added assertion in 2nd iteration.

---
In reply to: [114915361](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114915361) [](ancestors = 114915361)